### DISCUSSION TITLE
Adguard Home Does not Resolve Adguard/ on Unifi Setup

### DISCUSSION THREAD

**Discussion by stephhhen** [AdguardTeam/AdGuardHome#6452] [Q&A]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Custom (please mention in the description)

### Installation

Other (please mention in the description)

### Setup

On one machine

### AdGuard Home version

v0.107.41

### Action

```sh
nslookup -debug -type=a 'www.example.com' '$YOUR_AGH_ADDRESS'
```
```sh
Server:         10.0.1.101
Address:        10.0.1.101#53

------------
    QUESTIONS:
        google.ca, type = A, class = IN
    ANSWERS:
    ->  google.ca
        internet address = 142.251.33.163
        ttl = 300
    AUTHORITY RECORDS:
    ADDITIONAL RECORDS:
------------
Non-authoritative answer:
Name:   google.ca
Address: 142.251.33.163
```

I am running unifi locally and I could not access adguard via "adguard/" nor "adguard.local" , only thru the IP.

The LAN DNS is set to adguard's IP address to get all client name resolved. If I remove the LAN DNS setting then I could access "adguard/", but then all clients' requests will go under the router itself.

Note that I do have unbound setup for the DNS upstream. But the issue persists with/without unbound.

### Expected result

"adguard/" should be accesible.

### Actual result

"adguard/" and "adguard.local" refuses to connect when LAN DNS is set to adguard's address.

### Additional information and/or screenshots

Please let me know if more info is needed.

**Comment by ainar-g** [comment]
AdGuard Home itself doesn't create any sort of `adguard.local` domains, so you should probably get help from the people who support the Unifi package.  I'll transfer this issue to the Q&A discussions.
